MirahezeBots/plugins/wikimgnt.py

```diff
@@ -0,0 +1,136 @@
+"""wikimgnt.py - interact with Mediawiki API."""
+
+from MirahezeBots.utils import mwapihandler as mwapi
+
+from sopel.config.types import ListAttribute, StaticSection, ValidatedAttribute
+from sopel.module import commands, example
+
+
+class WikimgntSection(StaticSection):
```

Comment:
Let's use json for this. I know you started some work on wiki specific json. Could you restore that on top maybe?

Do you know how to write tests using pytest you could use for the config parsing?